components/Button/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, TouchableHighlight, View } from 'react-native';
+import { StyleSheet } from 'react-native';
+
+export default class Button extends React.Component {
+  static propTypes = {
+    action: PropTypes.string,
+    color: PropTypes.string,
+  }
+
+  getButtonStyle() {
```

Comment:
nice! maybe we can turn this into an arrow function :)

Reply:
I tried looking into it, but it seems that arrow functions do not return object literals.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Reply:
ohh okay np then! 👍